api/android/build-android-lib.sh

```diff
@@ -306,11 +306,15 @@ if [[ $enable_nnfw == "yes" ]]; then
         tar -zxf external/onert-ext-$nnfw_ver-android-aarch64.tar.gz -C external/nnfw
     fi
 
-    # Remove duplicated file c++shared.so
+    # Remove duplicated, unnecessary files (c++shared and tensorflowlite_jni)
     if [[ -e external/nnfw/lib/libc++_shared.so ]]; then
         rm external/nnfw/lib/libc++_shared.so
     fi
 
+    if [[ -e external/nnfw/lib/libtensorflowlite_jni.so ]]; then
+        rm external/nnfw/lib/libtensorflowlite_jni.so
+    fi
```

Comment:
I think you can just run `rm -f external/nnfw/lib/libtensorflowlite_jni.so`.

line 310-312 ditto

Reply:
Right!